frontend/src/metabase/query_builder/components/view/View/NotebookContainer/NotebookContainer.tsx

```diff
@@ -18,13 +20,22 @@ import { Flex, Box, rem } from "metabase/ui";
 // because we need to trigger the 'onTransitionEnd' in the component
 const delayBeforeNotRenderingNotebook = 10;
 
-interface NotebookContainerProps {
+type NotebookContainerProps = {
   isOpen: boolean;
-}
+} & NotebookProps;
 
 export const NotebookContainer = ({
   isOpen,
-  ...props
```

Comment:
`props` was being spread to `Notebook` so the only props that could be in here are `NotebookProps`